tempesta_fw/log.h

```diff
@@ -76,11 +76,19 @@
 #define TFW_WARN(...)	__CALLSTACK_MSG(KERN_WARNING TFW_BANNER		\
 					"Warning: " __VA_ARGS__)
 #define TFW_LOG(...)	pr_info(TFW_BANNER __VA_ARGS__)
+
+#define TFW_ERR_NL(...)		TFW_ERR(__VA_ARGS__)
```

Comment:
"NL" isn't obvious suffix, please leave a comment like `/* Not-limited printing for .... */`